apps/condo/domains/user/utils/directAccess/config.js

```diff
@@ -25,6 +25,7 @@ const DIRECT_ACCESS_AVAILABLE_SCHEMAS = {
     lists: [
         // Miniapps domain
         'B2BApp',
+        'B2BAppNewsSharingConfig',
```

Comment:
Please put it down a bit to keep order alphabetical?

Reply:
Done